_DeploymentAndDistroPackaging/ansible/roles/ciao-controller/tasks/create_cnci_image.yml

```diff
@@ -15,28 +15,28 @@
 
   - name: Check if cnci image is already setup
     become: no
-    stat: path=images/.cnci_ready
+    stat: path={{ workdir }}/.cnci_ready
     register: st
     changed_when: False
 
   - name: Download cnci image
     become: no
     get_url:
       url: "{{ cnci_image_url }}"
-      dest: images/{{ cnci_image }}.xz
+      dest: "{{ workdir }}/{{ cnci_image }}.xz"
```

Comment:
/tmp is often a tmpfs which means it is an ephemeral storage backed by RAM.  Many distros set it by default to be only a medium/small percentage of system RAM (eg: Ubuntu is 50% by default).  I worry storing the images there could amount to a DoS on the deployment machine by running it out of memory.